lib/rules/selector-max-id/index.js

```diff
@@ -12,7 +12,7 @@ const ruleName = "selector-max-id";
 
 const messages = ruleMessages(ruleName, {
   expected: (selector, max) =>
-    `Expected "${selector}" to have no more than ${max} id ${
+    `Expected "${selector}" to have no more than ${max} IDs ${
```

Comment:
`ID`.

Reply:
I intentionally switched to plural since that seemed to work better for any `${max}` > 1. Is `${max}` 1, most frequently, or do I misunderstand so that this better stays singular?

Reply:
Let's ask @jeddy3 or @ntwb, they know English better than I :)

Reply:
> Expected "${selector}" to have no more than _n_ IDs

With _n_ > 1 this is correct.

Reply:
> do I misunderstand so that this better stays singular?

The message, [like with other `selector-max-*` rules](https://github.com/stylelint/stylelint/blob/2d380692173fb9c67d566d440c6cba0fc33d3b4d/lib/rules/selector-max-attribute/index.js#L15-L20), is ended (via a ternary operator inside the template literal) with the word "selectors" if there is zero or two or more, or "selector" if there is one. This gives:

- Expected "${selector}" to have no more than 0 ID selectors
- Expected "${selector}" to have no more than 1 ID selector
- Expected "${selector}" to have no more than 2 ID selectors

As you can see, pluralising ID wouldn't make sense as it'll give:

- Expected "${selector}" to have no more than 0 IDs selectors
- Expected "${selector}" to have no more than 1 IDs selector
- Expected "${selector}" to have no more than 2 IDs selectors